epic-document-api/src/epic_document_api/resources/object.py

```diff
@@ -24,46 +23,52 @@
 from .apihelper import Api as ApiHelper
 
 
-API = Namespace('objects', description='Endpoints for Submission Management')
+API = Namespace("objects", description="Endpoints for Submission Management")
 """Custom exception messages
 """
 
 object_request_model = ApiHelper.convert_ma_schema_to_restx_model(
-    API, BlobObjectRequest(), 'An object to upload'
+    API, BlobObjectRequest(), "An object to upload"
 )
 
 object_response_model = ApiHelper.convert_ma_schema_to_restx_model(
-    API, BlobObject(), 'An object with auth headers'
+    API, BlobObject(), "An object with auth headers"
 )
 
 
-@cors_preflight('GET, OPTIONS, POST, DELETE')
-@API.route('', methods=['POST', 'GET', 'OPTIONS', 'DELETE'])
+@cors_preflight("GET, OPTIONS, POST, DELETE")
+@API.route("", methods=["POST", "GET", "OPTIONS", "DELETE"])
 class ObjectAuthHeaders(Resource):
     """Resource for managing objects s3 auth headers."""
 
     @staticmethod
-    @ApiHelper.swagger_decorators(API, endpoint_description='Get s3 auth headers for object')
+    @ApiHelper.swagger_decorators(
+        API, endpoint_description="Get s3 auth headers for object"
+    )
     @API.expect(object_request_model)
     @API.response(
-        code=HTTPStatus.OK, model=object_response_model, description='File with s3 auth headers'
+        code=HTTPStatus.OK,
+        model=object_response_model,
+        description="File with s3 auth headers",
     )
-    @API.response(HTTPStatus.BAD_REQUEST, 'Bad Request')
-    @cors.crossdomain(origin='*')
+    @API.response(HTTPStatus.BAD_REQUEST, "Bad Request")
+    @cors.crossdomain(origin="*")
     def post():
```

Comment:
is this deprecated now, should we call the "Get presigned urls" endpoint from now on?

Reply:
Let's keep the existing url for a while and replace it when we are feel okay with the new one